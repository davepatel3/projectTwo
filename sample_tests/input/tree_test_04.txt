string
peek_nodes
allocate
G*8Mr4tj`[Kr
0
ref_node
1
allocate
raF(H%{,ED
0
allocate
L<0xL[QArY}s
2
remove
2
allocate
K5@f.ld<P\
1
ref_node
3
allocate
yOd)n5a3EMY+aA
3
set_parent
2
1
set_parent
2
0
allocate
.e?cU*41*
0
remove
4
set_parent
3
0
allocate
V{os.:M
2
ref_node
2
ref_node
4
allocate
+{aVO=
0
remove
4
allocate
]44z-^X]G
2
remove
5
allocate
cQSJs1_:+g^an4>H
3
remove
1
remove
5
remove
4
ref_node
3
remove
3
remove
2
allocate
[n^xv%g}(
0
allocate
(^T*/~
1
ref_node
1
allocate
?2::Cz{x12w
1
allocate
r#qI$s@yW
5
remove
4
remove
1
peek_nodes
allocate
)P_<0isZ>Mt
0
remove
2
allocate
$)t:@fu
0
allocate
wkY8&LDdrkoem?E~
0
remove
3
allocate
n~{#i]3ve2Zcyzy
0
set_parent
4
5
allocate
0s,UPnM
0
allocate
.r^'*dr12Z$*
1
set_parent
4
2
allocate
vGM&[rN`
2
ref_node
5
remove
1
allocate
#p>^RPMZg,D`2CF{
5
ref_node
5
allocate
Zz\"l4+%kLZV
4
allocate
4O\ZxZ?
5
remove
2
remove
3
allocate
'*TnTH
0
allocate
aFNv*b
5
allocate
aq0fbn8%DYK1Ds?X
1
ref_node
1
allocate
AegdX6[,/b0(:;
5
remove
5
set_parent
3
0
allocate
EIKaC-x7G?'N*5GW
3
remove
3
allocate
~NX>d=l+'SKLb
1
peek_nodes
set_parent
2
0
allocate
+-D^
0
peek_nodes
set_parent
1
6
remove
2
ref_node
1
remove
6
allocate
lN!$&gXC>DN$WZx
0
remove
5
allocate
V(o-;]7=29=4(
0
ref_node
4
allocate
MIVA(wL"eg
0
allocate
q0~b"%u2eLh3d'm
3
set_parent
2
0
peek_nodes
remove
4
ref_node
3
allocate
3O/O
2
set_parent
1
3
ref_node
2
set_parent
2
1
peek_nodes
allocate
YLjyN!urC>
0
remove
2
ref_node
1
ref_node
1
allocate
-e`vBI=QMv
1
ref_node
5
set_parent
5
0
peek_nodes
allocate
8Qd7OAv
6
peek_nodes
allocate
ZfV%lZ#r*8m
5
allocate
_>Ae1<3G*Q.5K
0
remove
4
ref_node
1
remove
1
remove
3
ref_node
2
allocate
ZMs/8TVhp`Hsw2y_
2
allocate
u~B92P@""TyOj\
0
allocate
*1L+^'x{l~'A
4
remove
6
ref_node
4
allocate
l%00
7
set_parent
4
7
allocate
3J$R
2
allocate
!"K6q`eVHtgu6
5
peek_nodes
allocate
;X<cIp;?a[xJ
9
set_parent
10
8
allocate
i1)z)>!h\I
8
set_parent
8
0
ref_node
9
remove
7
remove
11
set_parent
1
8
ref_node
10
allocate
F:i1(;@"*&^X\Px
0
remove
2
allocate
sBfO@\B
9
allocate
iz-p>/TYMe
10
allocate
r!u=.!v0
5
set_parent
8
3
ref_node
3
allocate
&TLInD*;QpN6dq#t
9
remove
1
ref_node
9
allocate
w);lHT4,MT%xQH
4
ref_node
7
allocate
5VXHjkpy,c3
9
ref_node
7
allocate
$&{tb
2
remove
12
remove
7
allocate
5~A(@
2
remove
6
allocate
xkXLz$"
5
allocate
M${OZD$X/pm
2
remove
11
set_parent
2
0
allocate
\Vl#Ez$ES.e
3
set_parent
3
1
set_parent
8
11
ref_node
7
allocate
<Im*z
2
remove
10
allocate
pLJ;lzgB_-+ul
11
ref_node
3
set_parent
12
3
quit
