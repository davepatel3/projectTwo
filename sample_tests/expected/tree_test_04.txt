peek_nodes
0: 
allocate
1
ref_node
G*8Mr4tj`[Kr
parent: 0
children: 
allocate
2
allocate
3
remove
allocate
3
ref_node
K5@f.ld<P\
parent: 1
children: 
allocate
2
set_parent
set_parent
allocate
4
remove
set_parent
allocate
4
ref_node
yOd)n5a3EMY+aA
parent: 0
children: 4 
ref_node
V{os.:M
parent: 2
children: 
allocate
5
remove
allocate
4
remove
allocate
5
remove
remove
remove
ref_node
K5@f.ld<P\
parent: 0
children: 
remove
remove
allocate
1
allocate
5
ref_node
[n^xv%g}(
parent: 0
children: 5 
allocate
4
allocate
3
remove
remove
peek_nodes
0: 
1: [recycled]
2: [recycled]
3: [recycled]
4: [recycled]
5: [recycled]
allocate
2
remove
allocate
4
allocate
3
remove
allocate
5
set_parent
allocate
1
allocate
2
set_parent
allocate
3
ref_node
n~{#i]3ve2Zcyzy
parent: 0
children: 
remove
allocate
4
ref_node
n~{#i]3ve2Zcyzy
parent: 0
children: 4 
allocate
3
allocate
2
remove
remove
allocate
1
allocate
2
allocate
3
ref_node
'*TnTH
parent: 0
children: 3 
allocate
6
remove
set_parent
allocate
4
remove
allocate
2
peek_nodes
0: 
1: '*TnTH
2: ~NX>d=l+'SKLb
3: [recycled]
4: [recycled]
5: [recycled]
6: [recycled]
set_parent
allocate
6
peek_nodes
0: 
1: '*TnTH
2: ~NX>d=l+'SKLb
3: [recycled]
4: [recycled]
5: [recycled]
6: +-D^
set_parent
remove
ref_node
'*TnTH
parent: 6
children: 
remove
allocate
5
remove
allocate
4
ref_node
V(o-;]7=29=4(
parent: 0
children: 
allocate
3
allocate
2
set_parent
peek_nodes
0: 
1: [recycled]
2: q0~b"%u2eLh3d'm
3: MIVA(wL"eg
4: V(o-;]7=29=4(
5: [recycled]
6: [recycled]
remove
ref_node
MIVA(wL"eg
parent: 0
children: 
allocate
1
set_parent
ref_node
q0~b"%u2eLh3d'm
parent: 0
children: 
set_parent
peek_nodes
0: 
1: 3O/O
2: q0~b"%u2eLh3d'm
3: MIVA(wL"eg
4: [recycled]
5: [recycled]
6: [recycled]
allocate
6
remove
ref_node
3O/O
parent: 3
children: 
ref_node
3O/O
parent: 3
children: 
allocate
5
ref_node
-e`vBI=QMv
parent: 1
children: 
set_parent
peek_nodes
0: 
1: 3O/O
2: [recycled]
3: MIVA(wL"eg
4: [recycled]
5: -e`vBI=QMv
6: YLjyN!urC>
allocate
4
peek_nodes
0: 
1: 3O/O
2: [recycled]
3: MIVA(wL"eg
4: 8Qd7OAv
5: -e`vBI=QMv
6: YLjyN!urC>
allocate
2
allocate
7
remove
ref_node
3O/O
parent: 3
children: 
remove
remove
ref_node
ZfV%lZ#r*8m
parent: 5
children: 
allocate
4
allocate
1
allocate
3
remove
ref_node
ZMs/8TVhp`Hsw2y_
parent: 2
children: 3 
allocate
6
set_parent
allocate
8
allocate
9
peek_nodes
0: 
1: u~B92P@""TyOj\
2: ZfV%lZ#r*8m
3: *1L+^'x{l~'A
4: ZMs/8TVhp`Hsw2y_
5: -e`vBI=QMv
6: l%00
7: _>Ae1<3G*Q.5K
8: 3J$R
9: !"K6q`eVHtgu6
allocate
10
set_parent
allocate
11
set_parent
ref_node
!"K6q`eVHtgu6
parent: 5
children: 
remove
remove
set_parent
ref_node
;X<cIp;?a[xJ
parent: 8
children: 
allocate
6
remove
allocate
3
allocate
4
allocate
7
set_parent
ref_node
sBfO@\B
parent: 9
children: 8 
allocate
11
remove
ref_node
!"K6q`eVHtgu6
parent: 5
children: 3 11 
allocate
2
ref_node
r!u=.!v0
parent: 5
children: 
allocate
1
ref_node
r!u=.!v0
parent: 5
children: 
allocate
12
remove
remove
allocate
12
remove
allocate
7
allocate
6
remove
set_parent
allocate
11
set_parent
set_parent
ref_node
xkXLz$"
parent: 5
children: 
allocate
13
remove
allocate
4
ref_node
sBfO@\B
parent: 1
children: 11 
set_parent
quit
